src/main/java/jenkins/plugins/git/traits/WipeWorkspaceTrait.java

```diff
@@ -48,6 +49,7 @@ public WipeWorkspaceTrait() {
      * Our {@link hudson.model.Descriptor}
      */
     @Extension
+    @Symbol("gitWipeWorkspace")
```

Comment:
Would prefer that traits end with the suffix 'Trait' so that they don't prevent using that symbol for an extension that has the same name (like this one does).
```suggestion
    @Symbol("gitWipeWorkspaceTrait")
```

Reply:
Yet when I tried to write a test which used the symbol `stash` as proposed in the pull request, it failed immediately.  Are you confident that the symbols truly only need to be unique within the same extension point?

The `stash` symbol that was proposed is using the GitRepositoryBrowser extension point.  I'm reasonably confident that the `stash` symbol in pipeline is not using the GitRepositoryBrowser extension point.

Reply:
> The naming guidance is to avoid including part of the base class name in the symbol name, see here, so I would avoid adding a "Trait" suffix or similar. Symbols only need to be unique among the same extension point, so WipeWorkspaceTrait, which is an SCMSourceTrait, and WipeWorkspace, which is a GitSCMExtension, should be able to have the same symbol with no conflicts.

I'll take that as a worthwhile condition to test.  There is certainly an elegance to the idea that the AuthorInChangelog extension and the AuthorInChangeLog trait should use the same keyword, since they seem to only be usable within a context that is specific enough to disambiguate them by context.

Reply:
> Yet when I tried to write a test which used the symbol stash as proposed in the pull request, it failed immediately. Are you confident that the symbols truly only need to be unique within the same extension point?

Was the test for `stash` using a Pipeline, or CasC? I think `stash` will cause particularly unusual problems in a Pipeline because it clashes with the name of a step. Step function names are not symbols (they come from `StepDescriptor.getFunctionName`), so the problem should be of a different nature than regular symbol naming conflicts, but I am not sure exactly what would happen. Did you save the error message that you received from the test you performed?

Reply:
> Was the test for stash using a Pipeline, or CasC?

It was using a Pipeline step.  Since `stash` is a deprecated source provider that has been replaced by Bitbucket Server, let's leave the Stash class without a symbol and with a specific comment that it is intentionally without a symbol.

Reply:
This specific symbol, `wipeWorkspace` should be removed.  It is deprecated in Pipeline.  Pipeline should use `deleteDir()` as a general purpose Pipeline step instead of using the git plugin legacy extension `WipeWorkspace`.

Reply:
Cool. I'm going to:
* Remove symbols from `Stash` and `WipeWorkspace`. I'll include a comment which explains why for each of them.
* Revert [`814318d8`](https://github.com/kshultzCB/git-plugin/commit/814318d88fcec54cd3c39463de4c44095f33e6d6), per the discussion above about let's do some testing without the "Trait" suffix